packages/blockchain-wallet-v4-frontend/src/data/components/recurringBuy/selectors.ts

```diff
@@ -23,7 +24,13 @@ export const isAvailableMethod = (period: RecurringBuyPeriods, method?: SBPaymen
     if (!method) return false
     // All periods support ONE_TIME buys
     if (period === RecurringBuyPeriods.ONE_TIME) return true
-    return (paymentInfoPeriod && paymentInfoPeriod.eligibleMethods.includes(method.type)) || false
+    // Card type can be PAYMENT_CARD or USER_CARD, but backend doesn't support USER_CARD
+    // for recurring buy eligible payment methods. We may want to refactor cards code to
+    // remove the need for USER_CARD
```

Comment:
maybe to add follow up ticket for remove `USER_CARD`?

Reply:
Yeah I should document it a bit more, and fully understand how it's used. It has something to do when the card is created on the client but not fully created on the backend/everypay side.